multiversx_sdk/validators/validators_file.py

```diff
@@ -11,52 +12,57 @@
 
 
 class ValidatorsFile:
-    def __init__(self, validators_file_path: Path):
-        self.validators_file_path = validators_file_path
-        self._validators_data = self._read_json_file_validators()
+    def __init__(self, validator_signers: list[ValidatorSigner]):
+        self.signers = validator_signers
+
```

Comment:
In a future PR, let's:
 - rename class to `ValidatorsSigners` or `BunchOfValidatorsSigners` (or something similar)
 - rename `new_from_validators_file` to `new_from_json_bundle` (or something similar)

A separate idea: how about only supporting `new_from_pem` (multi-key PEM files)? Simpler & sufficient :thought_balloon: